docs/0.3/grid_purchase_order.md

```diff
@@ -1,158 +1,62 @@
+
+## Background
+
+Today’s supply chains are complex, which makes managing them difficult and
+expensive. Some of this complexity comes from the volume and intrinsic
+complexity of the transactions and relationships between buyers and sellers.
+This is often compounded by buyers and sellers maintaining their own set of
+purchase order records.
+
+Separate record keeping creates two primary problems: it creates disputes when
+the parties do not agree, and it imposes a cost on both parties to maintain
+their systems in an agreeing state.
+
+Grid Purchase Order solves the problem of separate record keeping, thereby
+reducing costly disputes between parties and eliminating multi-system
+maintenance activities. Organizations can use Purchase Order either as a
+system of record or as a bridge between their and their business partners’
+existing systems of record.
+
+Purchase Order enables all parties in a private Splinter circuit or Sawtooth
+blockchain to see the same data and receive updates in near-real time. Unlike
+centralized databases, which can achieve a similar result, Grid gives equal
+standing to all parties in the circuit by using distributed ledger technology
+(DLT) to ensure data integrity in a decentralized way. It also increases
+reliability by providing each party with their own copy of the data.
+
+## How Purchase Order works
+
+Purchase Order is a smart contract that leverages the Grid Workflow smart
+contract to model the business processes that surround the real-world purchase
+order process. Organizations can use pre-built workflows or create their own
+to fit their needs.
+
+Purchase Order workflows use three core components to facilitate transactions:
+- workflow states - these represent the steps of the business process
+- constraints - these represent business rules that determine when the 
+  purchase order can move between workflow states
+- permissions - these define the actions that each organization can take at
+  each step of the purchase order process (that is, at each workflow state)
+
+Each purchase order contains one or more versions. Organizations can propose
+or consider multiple versions of a purchase order before finalizing one.
+
+Each purchase order version is comprised of one or more revisions. Each change
+to a version’s content creates a new revision, providing users with a revision
+history for every version.
+
+## How we expect Purchase Order will be used
+
+As mentioned above, organizations may adopt Grid Purchase Order as the system
+of record between them and their partner organizations. In this case, their
+technology integration teams will likely enable users to interact with Grid
+through some integration application.
+
+Organizations that choose to use their existing systems of record can use Grid
+Purchase Order to synchronize the purchase order records of them and their
+partners. In this case, their technology teams will integrate Grid directly
+with their systems of record.
```

Comment:
I think at some point in this doc, we should also mention the XML validation similar to how we mention this for the product feature. Definitely something we should highlight here, moreso the part of using GS1 standards for the order XML

Reply:
That could probably go in the "how this works" section

Reply:
Added